src/foam/nanos/approval/ApprovalRequestNotificationNotificationCitationView.js

```diff
@@ -163,12 +164,19 @@ foam.CLASS({
       text-align: center;
       float: right;
       margin-right: 145px;
-      margin-top: 14px;
     }
     ^ .userSummaryDiv {
       position: relative;
-      top: 8;
       display: inline-block;
      }
+   ^ .p {
```

Comment:
Why do we need these?

Reply:
To fix css between classification and description, sometimes it looks like... 

<img width="1115" alt="Screen Shot 2022-03-22 at 2 33 25 PM" src="https://user-images.githubusercontent.com/33228583/159551659-3351029a-efb5-4869-bc52-173bdfda8ab4.png">